### DISCUSSION TITLE
Net5 Using Observable.FromEventPattern<KeyEventArgs>

### DISCUSSION THREAD

**Discussion by SID9-HoneyBee** [dotnet/reactive#1599] [Q&A] [answered]
I want to use RX in WPF Net5.

To control the KeyDown event
I used Observable.FromEventPattern<KeyEventArgs> .

I installed System.Reactive as a Nuget package, but

ObserverOn does not find DispatcherScheduler .

keyDownSubscripton = Observable.FromEventPattern<KeyEventArgs>(this, "KeyDown")
            .Do(e =>
            {
                // Do something on key down event

            })
            .Delay(TimeSpan.FromMilliseconds(10))
            .ObserveOn(System.Reactive.Concurrency.DispatcherScheduler.Current) // OnNext on UI thread
            .Subscribe(e =>
            {
                // Do something after delay
                //viewModel.ChangeContentCommand.Execute(null);
            });

How do I fix it?

![image](https://user-images.githubusercontent.com/76090330/131064504-1306da03-59e2-4c66-b4f8-5c13b66f71a6.png)

**Comment by clairernovotny** [comment, marked as answer]
You need to specify the 19041 SDK, like this `net5.0-windows10.0.19041` as the TFM. You can target lower builds of Windows if desired, as that simply specifies the SDK used to build.
